Source/SwiftLintFramework/Baseline/Baseline.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+public class Baseline {
+    private let baselinePath: String
+    private var baselineViolations = [BaselineViolation]()
+
+    public init(baselinePath: String) {
+        self.baselinePath = baselinePath
+    }
+
+    public func isInBaseline(violation: StyleViolation) -> Bool {
+        let baselineViolation = BaselineViolation(
+                ruleIdentifier: violation.ruleIdentifier,
+                location: violation.location.description,
+                reason: violation.reason
+        )
+        let contains = baselineViolations.contains(baselineViolation)
+        return contains
+    }
+
+    public func saveBaseline(violations: [StyleViolation]) {
+        let fileContent = violations.map(generateForSingleViolation).joined(separator: "\n")
```

Comment:
would \n work on all platform ? trying point out things . like when files gets modified in git or in other os